AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/edit-work-hours/vho-work-hours-non-availability-table/vho-work-hours-non-availability-table.component.ts

```diff
@@ -36,6 +36,7 @@ export class VhoWorkHoursNonAvailabilityTableComponent implements OnInit, CanDea
         });
     }
     @Input() set result(value) {
+        this.hideMessage();
```

Comment:
have you used this new method in all the other part of the component? We have displayMessage = false in different part of it

Reply:
good point - updated